vello_encoding/src/glyph_cache.rs

```diff
@@ -245,13 +245,19 @@ pub(crate) struct HintKey<'a> {
 
 impl<'a> HintKey<'a> {
     fn instance(&self) -> Option<HintingInstance> {
-        HintingInstance::new(self.outlines, self.size, self.coords, HINTING_MODE).ok()
+        HintingInstance::new(self.outlines, self.size, self.coords, HINTING_OPTIONS).ok()
     }
 }
 
-const HINTING_MODE: HintingMode = HintingMode::Smooth {
-    lcd_subpixel: Some(LcdLayout::Horizontal),
-    preserve_linear_metrics: true,
+const HINTING_OPTIONS: HintingOptions = HintingOptions {
+    engine: skrifa::outline::Engine::AutoFallback,
+    target: skrifa::outline::Target::Smooth {
+        mode: skrifa::outline::SmoothMode::Lcd,
+        // TODO: Set based on whether the anti-aliasing mode is Area?
+        symmetric_rendering: false,
+        // TODO: Does Parley handle layout in a hinting-aware way?
```

Comment:
Parley expects linear metrics. Hinting aware layout requires scaling glyph outlines which is incredibly expensive.